alpha-lab/phone-neon/index.md

```diff
@@ -1,118 +1,103 @@
 
-So far, we've only scratched the surface by examining static images. Now, it's time to dive into the dynamic world of
-our smartphones and explore gaze behaviour more naturally while scrolling. Use the dynamic screen mapping tutorial for this one!
+As shown in this example, there are clear differences between the areas of interest in both attentional allocation and salience that became evident thanks to the quantitative analysis that focused on the measures mentioned earlier. In line with the heatmap and scanpath visualizations, it is evident that dwell time is higher in AOI-1 compared to the other two areas of interest. Additionally, the time to first contact metric revealed that users directed their attention more swiftly towards AOI-0.
 
-Checking out the recording, Neon's accuracy richly captures gaze behaviour and provides a nice high-level overview of
-what the wearer was looking at.
+For a detailed explanation and visual representation of these metrics, you can refer to our [AOI tutorial](/gaze-metrics-in-aois/), which provides step-by-step instructions and charts for your analysis.
 
-Visualisations are great, but the real power of this tool is that it generates a CSV file containing gaze data mapped
-onto the screen, in 2D x, y coordinates. This offers many possibilities for further customisation and in-depth analysis.
+### Map Your Gaze Onto Dynamic Phone Screen Content
 
 <Youtube src="RKrf3YQjzao"/>
 
-## In the wild
+Until now, the tools that we have used have been limited to mapping gaze onto static images. However, the dynamic realm of mobile phones makes it important to study gaze behaviour in a more natural context, particularly during scrolling. We, therefore, used the [dynamic screen mapping tutorial](/map-your-gaze-to-a-2d-screen/) for this purpose. This approach provided a more comprehensive understanding of gaze behaviour in dynamic interfaces.
 
-That's all fine, but what does it look like to interact with phone screens when out and about? Consider this scenario:
-Imagine searching for products on your mobile app while looking at the physical products on a supermarket shelf. We
-brought Neon into the wild to assess its performance beyond controlled environments. See an example of real-world user
-behaviour below!
+Upon reviewing the recording, Neon demonstrated good accuracy in capturing gaze behaviour, offering a useful high-level overview of the wearer's focal points.
 
-<Youtube src="enkOC7_wf0U"/>
+Another very useful facet of this tool is its ability to generate a CSV file containing gaze data mapped onto the screen, represented in 2D x, y coordinates. This feature opens avenues for extensive customisation and more in-depth analysis, which we don’t cover here.
```

Comment:
Can we make this more concise?

Reply:
What do you think about something like that: "Another very useful facet of this tool is its ability to generate a CSV file containing gaze data mapped onto the screen, represented in 2D x, y coordinates. Although this feature is beyond the scope of the present article, it opens avenues for extensive customisation and more in-depth analysis, such as creating dynamic heatmaps overlaid on the screen recording."

Reply:
Is there anywhere we could link out to for this and/or just condense into a single sentence?

Reply:
This sentence refers to the csv output of the Dynamic RIM article on Alpha Lab. The link to this article has been already added in this section. However, there isn't a dedicated section for the CSV file output of Dynamic RIM in the article, so there's no specific link for that. A more condensed version in one sentence could be: "This tool generates a CSV file with 2D x, y coordinates for gaze data on the screen, enabling extensive customisation and in-depth analysis, though beyond this article's scope." What do you think?